src/platform/silabs/SiWx917/SiWxPlatformInterface.h

```diff
@@ -59,6 +59,14 @@ void gpio_uulp_pin_interrupt_callback(uint32_t pin_intr)
     }
 }
 
+/**
+ * @brief Processing function when a button is triggered
+ *
+ * @param btn which button was pressed
+ * @param btnAction the action that triggered the buttone vent
+ */
+void sl_button_on_change(uint8_t btn, uint8_t btnAction);
```

Comment:
not a strong ask, but this should probably be on the wifi SPAM section. This comes from sisdk for efr32